src/types/json.h

```diff
@@ -217,6 +217,42 @@ struct JsonValue {
     return results;
   }
 
+  StatusOr<std::vector<std::string>> GetBytes(std::string_view path, JsonStorageFormat format,
+                                              int max_nesting_depth = std::numeric_limits<int>::max()) const {
+    std::vector<std::string> results;
+    try {
+      jsoncons::jsonpath::json_query(value, path, [&](const std::string & /*path*/, const jsoncons::json &origin) {
+        if (!origin.empty()) {
```

Comment:
What's the meaning of this empty check?
Could you think about whether an empty JSON array/object will occupy storage space?